src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -470,6 +509,10 @@ public unsafe void Initialize()
                 _initialOptions.API.Profile == ContextProfile.Core ? OpenGlProfile.Core : OpenGlProfile.Compat
             );
 
+            // Set video mode
+            _glfw.WindowHint(WindowHintInt.RefreshRate, _initialOptions.VideoMode.RefreshRate);

```

Comment:
Oh shoot didn't realize you can actually change the refresh rate. **Ignore my previous comments on removing VideoMode from IViewProperties!!!!!** Instead, should we move `DepthBufferBits` to `VideoMode` or is this not applicable?

Reply:
It was part of video mode in my first try but it isn't applicable imho. You can't change it nor will it have any influence after window creation. Moreover it has nothing to do with video mode. Instead it is just a hint for the window to support an appropriate context.

Reply:
Alright :+1:

Reply:
The problem I see is that a user would expect the given depth buffer bits to be active but glfw may ignore this or choose a different bit depth if the given one is not possible. The opengl context is imho the only source which could tell if the given depth buffer bits were used. Any suggestions how we could improve user experience here?